tests/integration/helpers.py

```diff
@@ -29,6 +29,7 @@
 METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
 APP_NAME = METADATA["name"]
 
+EXPECTED_SNAP_REVISION = 37
```

Comment:
just to simplify future changes, I would import revision directly from `src/opensearch.py`.
something like
```
from opensearch import OPENSEARCH_SNAP_REVISION
```
but, it is not blocker

Reply:
Hi @delgod I am not sure that works any longer with the new CI, but I will try it out and make a PR for that